pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md

```diff
@@ -18,7 +18,7 @@ After adopting the cluster, you use this AMI to scale up, or replace a failed in
 
 <p class="message--note"><strong>NOTE: </strong>Konvoy v1.8.4 uses Kubernetes version 1.20.13 by default. If upgrading from Konvoy v1.8.3, use Kubernetes version 1.20.11 for the below commands.</p>
 
-1.  Create the AMI using [Konvoy Image Builder][kib] (you can [download the newest release on GitHub][kib-releases]):
+1.  Create the AMI using [Konvoy Image Builder][kib] (you must [use the Konvoy Image Builder v1.5.0 release, downloadable from GitHub][kib-releases]):
```

Comment:
I think they should be able to use both 1.5.0 1.7.0

if that's the case, should we just tell them to use 1.7.0